jekyll/_cci2/hello-world.adoc

```diff
@@ -0,0 +1,531 @@
+---
+redirect_from: /examples-intro/
+tags:
+  platform:
+    - Cloud
+    - Server v4.x
+    - Server v3.x
+    - Server v2.x
+sectionTags:
+  Cloud:
+    - "#echo-hello-world-cloud"
+  Server_v4:
+    - "#echo-hello-world-server-4"
+  Server_v3:
+    - "#echo-hello-world-server-3"
+  Server_v2:
+    - "#echo-hello-world-server-2"
+---
+= Hello world
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Get started with CircleCI by printing Hello World in any execution environment.
+:icons: font
+
+This page describes how to get started with a basic pipeline using any execution environment. The steps described here assume you want to create a new project locally and push your changes to your VCS using `git`. 
+
+As an alternative, if you already have a repository you want to "plug in" to CircleCI you can set this up entirely within the CircleCI web app, including commiting a config file. This flow is outlined in the link:/docs/getting-started[Quickstart guide].
+
+This page assumes you have already link:/docs/first-steps/[signed up] for CircleCI.
+
+[#echo-hello-world-cloud]
+== Echo hello world
+
+This example adds a job called `hello-job` that simply prints `hello world` to the console. To get started, complete the following steps:
+
+. Create a directory called `.circleci` in the root directory of your local code repository.
+
+. From the `.circleci` directory you just created, create a link:/docs/configuration-reference[`config.yml`] file and add the following content (select your execution environment using the tabs):
++
+[.tab.hellocloud.Docker]
+--
+The job `hello-job` spins up a container running a link:/docs/circleci-images/#nodejs[pre-built CircleCI Docker image for Node]. Refer to link:/docs/using-docker[Using the Docker execution environment] page for more information.
```

Comment:
Is there a way to leave a bit more spacing between the tabs and the text within each tab? The GPU one looks fine to me but all the others look a bit squished.

The text wrapping in the tabs also looks a bit off. When I have my browser at a certain smaller size, the sentence "The job `hello-job`..." doesn't include the period in the wrapping so it shows up in the next line, separated from the sentence.

Reply:
Yes the spacing between tab and text is something I've asked DD about before, I've just asked again. I'll ask about wrapping too :-D